tests/Builder/BuilderEncoderTest.php

```diff
@@ -279,6 +279,60 @@ public function testRedactStage(): void
         $this->assertSamePipeline($expected, $pipeline);
     }
 
+    public function testUnwind(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::unwind('$foo'),
+            Stage::unwind('$foo.bar'),
+            Stage::unwind(Expression::arrayFieldPath('foo.bar.baz')),
+        );
+
+        $expected = [
+            ['$unwind' => '$foo'],
```

Comment:
This is the short syntax of [`$unwind`](https://www.mongodb.com/docs/manual/reference/operator/aggregation/unwind/). In order to cover all use cases while keeping the code as simple as possible, I've chosen not to detect when only the field path is provided. So it's always an object with at least one property "path" that's returned.

```suggestion
            ['$unwind' => ['path' => '$foo']],
```

Reply:
Ok it would be helpful if the diff would show this when failing

Reply:
Using JSON comparison, the error of looks like this:

```diff
1) MongoDB\Tests\Builder\BuilderEncoderTest::testUnwind
'{ "root" : [ { "$unwind" : { "path" : "$foo" } }, { "$unwind" : { "path" : "$foo.bar" } }, { "$unwind" : { "path" : "$foo.bar.baz" } } ] }'
Failed asserting that '{ "root" : [ { "$unwind" : { "path" : "$foo" } }, { "$unwind" : { "path" : "$foo.bar" } }, { "$unwind" : { "path" : "$foo.bar.baz" } } ] }' matches JSON string "{ "root" : [ { "$unwind" : "$foo" }, { "$unwind" : "$foo.bar" }, { "$unwind" : "$foo.bar.baz" } ] }".
--- Expected
+++ Actual
@@ @@
 {
     "root": [
         {
-            "$unwind": "$foo"
+            "$unwind": {
+                "path": "$foo"
+            }
         },
         {
-            "$unwind": "$foo.bar"
+            "$unwind": {
+                "path": "$foo.bar"
+            }
         },
         {
-            "$unwind": "$foo.bar.baz"
+            "$unwind": {
+                "path": "$foo.bar.baz"
+            }
         }
     ]
 }

```